docs/spec-v1/concepts/perspectives.md

```diff
@@ -22,12 +22,30 @@ Static metadata is known at design-time or deploy-time, when a new version of an
 For cloud software with continuous delivery, the version may not be explicit or of interest to the consumer, a fallback to a "latest" version may be needed.
 But consider that also cloud software that is going through phased deployments therefore can have multiple versions active at the same time.
 
-The static perspective describes the shared metadata of all system instances (tenants) of the same system version.
+In the case that the system is not versioned (continuous delivery) or resources do not relate to a system directly, the `system-type` perspective can be used.
+The `system-type` information apply no matter which system version or instance is used.
+
+The static perspective describes the shared metadata of all system instances (tenants) of the same system version or version independent.
 Either the metadata is always the same or it explicitly ignores the tenant-specific extensibility, configuration and any feature toggles - describing what's generic and shared.
 The advantage of static metadata is that is always available, there is no need to first provision tenants to get it. It is also a good integration contract for everything that is meant to work potentially with _any_ tenant.
 
 At SAP, we have the [SAP Business Accelerator Hub](https://api.sap.com/) that documents the static perspective.
 
+#### System Version Guidelines
```

Comment:
@swennemers  please review - is this also how you would propose it?

I'm not sure if we need an implied "latest" anymore (if system-version perspective, but no version is given). I'm afraid that starting with "latest" will become complicated quickly, so let's avoid it if not needed.